docs-src/cloud/tcld/user/set-account-role.md

```diff
@@ -0,0 +1,64 @@
+---
+id: set-account-role
+title: tcld user set-account-role
+sidebar_label: set-account-role
+description: How to set an account-level Role for a user in Temporal Cloud using tcld.
+tags:
+  - tcld
+---
+
+The `tcld user set-account-role` command sets an [account-level Role](/cloud/#account-level-roles) for the specified user in Temporal Cloud.
+You must set either `--user-email` or `--user-id`.
+
+Alias: `ri`
+
+The following modifiers control the behavior of the command.
+
+### `--user-email`
+
+Specify the email address of the user to assign an account-level Role to.
+
+Alias: `-e`
+
+**Example**
+
+```command
+tcld user set-account-role --user-email <test@example.com> --account-role Developer
+```
+
+### `--user-id`
+
+Specify the user identifier of the user to assign an account-level Role to.
+
+Alias: `--id`
+
+**Example**
+
+```command
+tcld user set-account-role --user-id <test-user-id> --account-role Developer
+```
+
+### `--request-id`
+
+The request identifier to use for the asynchronous operation.
+
+If not set, the server assigns an identifier.
+
+Alias: `-r`
+
+### `--resource-version`
```

Comment:
`--resource-version` is also supported in `set-namespace-permissions` and `delete`.